internal/conduit/registry/geth_block_test.go

```diff
@@ -11,45 +11,30 @@ import (
 	"github.com/base-org/pessimism/internal/conduit/pipeline"
```

Comment:
@anupsv-cb Any chance you could look over this file?

Reply:
Definetely.